packages/cli/src/lib/setups/i18n/replacers.test.ts

```diff
@@ -96,11 +86,8 @@ describe('i18n replacers', () => {
           /**
            * Declare local additions to the Remix loader context.
            */
-          interface AppLoadContext {
+          interface AppLoadContext extends HydrogenContext {
             env: Env;
-            cart: HydrogenCart;
-            storefront: Storefront<I18nLocale>;
-            customerAccount: CustomerAccount;
```

Comment:
The correct test here is matching `HydrogenContext<I18nLocale>`, otherwise the code is not doing what we want.

Reply:
Took me a little while to figure out how to use https://ast-grep.github.io/playground.html and write AST with it, but I fixed the replacer now and this was really fun!